spec/unit/concerns/api_spec.rb

```diff
@@ -427,6 +427,36 @@
     end
   end
 
+  describe '.get_updated_between' do
+    let(:sobject) {'Account'}
```

Comment:
Can we add spaces inside the curly braces? (e.g. `{ 'Account' }`) Here and elsewhere!